server/@types/express/index.d.ts

```diff
@@ -20,6 +20,11 @@ export declare global {
       verified?: boolean
       id: string
       logout(done: (err: unknown) => void): void
+      session: {
+        returnTo: string
+        nowInMinutes: number
```

Comment:
not quite sure why the type check works for the `server` directory without this, but doesn't do any harm adding it anyway